operations/power_management/Shut_Down_and_Power_Off_the_Management_Kubernetes_Cluster.md

```diff
@@ -86,7 +87,15 @@ documentation (`S-8031`) for instructions on how to acquire a SAT authentication
 
 ## Shut down the Kubernetes management cluster
 
-1. (`ncn-mw#`) Shut down platform services.
+1. (`ncn-m001#`) Set variables for the three types of management NCNs as comma-separated lists.
```

Comment:
Set variables for the three types of management NCNs as comma-separated lists.
--> Set variables as comma-separated lists for the three types of management NCNs.